packages/ui/src/watch/utils.ts

```
@@ -0,0 +1,16 @@
+export const formatCreatedDate = (date: string | null | undefined): string => {
+  if (!date) {
+    return '';
+  }
+  try {
+    const parsedDate = new Date(date);
+    if (isNaN(parsedDate.getTime())) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)